Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -46,6 +50,45 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager) {
         masterTalons = new HashMap<XCANTalon, BaseDriveSubsystem.MotionRegistration>();
         masterTalons.put(leftMaster, new MotionRegistration(0, 1, -1));
         masterTalons.put(rightMaster, new MotionRegistration(0, 1, 1));
+        
+        
+        leftMaster.createTelemetryProperties("LeftDriveMaster");
+        rightMaster.createTelemetryProperties("RightDriveMaster");
+        
+        leftTicksPerFiveFeet = propManager.createPersistentProperty("leftDriveTicksPer5Feet", 0);
+        rightTicksPerFiveFeet = propManager.createPersistentProperty("rightDriveTicksPer5Feet", 0);
+        
+    }
+    
+    /**
+     * Helper function to get the "typical" number of inches for a given number of ticks. This assumes
+     * that there isn't much difference between left/right drive encoders.
+     * @param ticks Number of encoder ticks
+     * @return Number of inches
+     */
+    public double ticksToInches(double ticks) {
+        double ticksPerInch = getAverageTicksPerInch();
+        
+        if (ticksPerInch != 0) {
+            return ticks / ticksPerInch;
+        }
+        return 0;
+    }
+    
+    /**
+     * Helper function to get the typical number of ticks for a given number of inches. This assumes
+     * that there isn't much difference between left/right drive encoders.
+     * @param inches Number of inches
+     * @return Number of encoder ticks
+     */
+    public double getInchesToTicks(double inches) {
```

Comment:
TBH, I'm still a bit confused by why we want to average them; why not have a `getLeftTicksPerInch` and `getRightTicksPerInch`? I would expect that if the goal is to get the average distance travelled it would be done within this class rather than externally.

Reply:
I think your approach is better. I initially thought some external consumer might want to request motion in generic distance units, but we should probably keep things specific.